### PR TITLE
Depthmap toolkit refactor

### PR DESCRIPTION
**Pull request by lvonasek**
# Description

This PR implements multiple minor fixes, speed improvement and mainly RGB-D flip fix of the data:
* Added flag original_orientation in toolkit.py, so it is easy to show the data in original orientation
* Fixed next, prev button, broken by #436
* Fixed face bluring (it was grayscaled),  broken by #426, also adjusted constant for face bluring size
* Speedup segmentation (using np-arrays), improve floor segmentation (less random pixels)
* L-R flipping of depthmap is now done in parsing, not visualisation (by this RGB-depth-IR_confidence array has the same coordinate system). @markus-hinsche could you check your RGB-D models if there are any inconsistences? (I am afraid there might RGB and other arrays have different L-R flipping)

# How Has This Been Tested?
- [x] Check if the values from clicking on depthmap are correct
- [x] Check if exported OBJ has correct dimensions
- [x] Visual check that all visualisation has the same orientation
- [x] Visual check that the visualisation are not flipped
- [x] Visual check OBJ, PCD export
- [x] Visual check of mulitple OBJ exports shares the coordinate system
- [x] Run CV evaluation if the segmentation modification affect the results
- [x] Run CV renderer for all good and bad scans

CV evaluation before:
Processed scans 670
Rejected scans 636
Process time 22 minutes
Average error: 2.104 cm

CV evaluation after:
Processed scans 671
Rejected scans 635
Process time: 6 minutes
Average error: 2.100 cm

### CODE REVIEW

File: cgmml/common/depthmap_toolkit/visualisation.py [hunk 1]
```diff
@@ -193,10 +166,9 @@ def render_plot(dmap: Depthmap) -> np.ndarray:
         render_confidence(dmap),
     ]
     if dmap.has_rgb:
+        highest_point: np.ndarray = dmap.get_highest_point(mask)
         output_rgb = render_rgb(dmap)
         output_rgb = blur_face(output_rgb, highest_point, dmap)
         output_plots.append(output_rgb)
 
-    output = np.concatenate(output_plots, axis=1)
-    logger.info('height=%fm', highest_point[1] - floor)
```
Comment (markus-hinsche) [inline cgmml/common/depthmap_toolkit/visualisation.py:201]: Why not output this anymore?

**Review by markus-hinsche — COMMENTED** [review]
LGTM, very nice refactoring. Lots of simplifications. 

I think we should rework `original_orientation` as it turns weirdly, see comment. 

Thanks for fixing bugs. In order to prevent bugs in the future, we could add additional unit tests to check those things fixed here so they can't degrade into bugs again

**Comment by markus-hinsche** [comment]
I check that RGB+depth by looking and verifying the outputs of test_artifact_processor_rgbd again. This looks correct, same as in the depthmap toolkit
image

File: cgmml/common/depthmap_toolkit/visualisation.py (hunk 1 above)
Comment (lvonasek) [reply cgmml/common/depthmap_toolkit/visualisation.py:201]: It costs time to calculate highest_point. It slows down the renderer.py from CV evaluation a lot.

File: cgmml/common/depthmap_toolkit/toolkit.py [hunk 2]
```diff
@@ -88,7 +89,10 @@ def show(depthmap_dir: str, calibration_file: str):
     angle = DMAP.get_angle_between_camera_and_floor()
     logging.info('angle between camera and floor is %f', angle)
 
-    plt.imshow(render_plot(DMAP))
+    output = render_plot(DMAP)
+    if original_orientation:
+        output = ndimage.rotate(output, 90)
```
[the remark this answers is not in the document]
Comment (lvonasek) [reply cgmml/common/depthmap_toolkit/toolkit.py:94]: It is only for debug purposes, I do not want to put `original_orientation` into `visualisation.py` - that would make the interface unnecessary difficult

**Pull request merged**